conf/messages

```diff
@@ -55,6 +55,17 @@ checkingRecords.ffid=Identifying file formats
 fileChecksResults.header=Record check results
 fileChecksResults.title=Record check results
 fileChecksResults.description=Here are your results
+forbiddenError.header=You are not permitted to see this page
+forbiddenError.body=This page is restricted to certain users only.
+internalServerError.header=Sorry, there is a problem with the service
+internalServerError.body=Try again later.
+unauthenticatedError.header=You must be logged in to see this page
+unauthenticatedError.body=Please <a href= "#">login</a> to continue.
+notFoundError.header=This page doesn't exist
+notFoundError.body1=If you typed a web address, check it is correct.
+notFoundError.body2=If you pasted the web address, check you copied the entire address.
+notFoundError.body3=If the web address is correct or you selected a link or button, contact <a href= "#">The National Archives Help Desk<a/> to speak to someone about your transfers.
+notFoundError.body4=Alternatively, you can <a href= "/dashboard">return</a> to the home page.
```

Comment:
Should the link be on the whole "return to the home page" phrase rather than just "return"? This is good for accessibility reasons (see https://www.washington.edu/accessibility/links/ for example) and I think if I was the user, I'd expect to be able to click anywhere in that phrase.

Reply:
Oh true! I've changed that now.